src/qt/rpcconsole.cpp

```diff
@@ -175,6 +175,11 @@ bool RPCConsole::RPCParseCommandLine(std::string &strResult, const std::string &
             nDepthInsideSensitive = 1;
             filter_begin_pos = chpos;
         }
+        // Make sure stack is not empty before adding something
+        if(stack.empty()) 
```

Comment:
space after `if` please and left brace on the same line (or maybe one line `if` without braces - see https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md).